rw-system.sh

```diff
@@ -69,7 +69,7 @@ changeKeylayout() {
     if getprop ro.vendor.build.fingerprint | grep -iq \
         -e xiaomi/polaris -e xiaomi/sirius -e xiaomi/dipper \
         -e xiaomi/wayne -e xiaomi/jasmine -e xiaomi/jasmine_sprout \
-        -e xiaomi/platina -e iaomi/perseus -e xiaomi/ysl \
+        -e xiaomi/platina -e xiaomi/perseus -e xiaomi/ysl \
```

Comment:
that's not a typo, but whatever

Reply:
iaomi isn't a typo? Ew

Reply:
because `Xiaomi` and `xiaomi`, seems specific for BSP devices and ODM devices, and `grep` just match keywords, and the keyword `iaomi` is not affected for matching device.

Reply:
You're passing -i to it in the beginning.